OpenSearch: Use favicon if available

Blendify opened the pull request:
Subject: OpenSearch: Use favicon if available

### Feature or Bugfix
- Feature

### Purpose
- Better support for OpenSearch

### Detail
- Instead of users having to use templates to add an `<Image>` this can be done by using the favicon if it exists.

### Relates
- None

tk0miya committed:
refactor quickstart

tk0miya committed:
Make EventManager portable

So far, we need to bypass application object for modules to emit
a event.  This make EventManager portable and easy to pass event
emitter.  This brings modules less coupled with application object.

runawayhorse001 committed:
Update Example

tk0miya committed:
Bump version

tk0miya committed:
Drop features and APIs deprecated in 1.8

runawayhorse001 committed:
Update EXAMPLES

White space added before <

tk0miya committed:
Merge pull request #6219 from runawayhorse001/patch-1

Update Example

tk0miya committed:
Merge pull request #6223 from tk0miya/RemovedInSphinx30Warning

Drop features and APIs deprecated in 1.8

tk0miya committed:
Merge pull request #6233 from tk0miya/refactor_quickstart

refactor quickstart

tk0miya committed:
Add RemovedInSphinx50Warning

tk0miya committed:
Merge pull request #6259 from tk0miya/deprecation

Add RemovedInSphinx50Warning

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Fix flake8 violations

jfbu committed:
Partially revert #6232, synchronize new Makefile with old (refs: #4092)

tk0miya committed:
Merge branch '2.0'

jfbu committed:
Merge pull request #6303 from jfbu/6232_partial_revert

Partially revert #6232, synchronize new Makefile with old (refs: #4092)

jfbu committed:
Update CHANGES for #6303

tk0miya committed:
refactor: Make app argument for EventManager optional to keep compatibility

tk0miya committed:
Merge pull request #6260 from tk0miya/refactor_events

Make EventManager portable

jfbu committed:
Make imgmath_use_preview work also for svg output

jfbu committed:
Merge branch '2.0'

jfbu committed:
Merge branch '2.0'

jfbu committed:
More robust code for getting saved "depth" of svg inline math snippet

jfbu committed:
Reorganize documentation of the imgmath configuration variables

jfbu committed:
Merge pull request #6310 from jfbu/imgmath_correct_svg_baselines

Make imgmath_use_preview work also for svg output

jfbu committed:
Update CHANGES for PR #6310

juh2 committed:
adding German book about Sphinx

Adding a link to the authors bookpage.

SuryaGrandhi committed:
Removing broken links

Using sphinx with Read the Docs, the Getting Started link has been updated.

tk0miya committed:
Merge pull request #6334 from SuryaGrandhi/patch-1

Removing broken links

tk0miya committed:
Merge pull request #6333 from juh2/patch-1

adding German book about Sphinx

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Fix flake8 violation

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
doc: Fold lines by 80 columns

jia-kai committed:
Fix the ranking of search results: object name match should be computed based on lower-case fullname

gilch committed:
Grammar corrections

tk0miya committed:
Merge pull request #6414 from gilch/patch-1

Grammar corrections

tk0miya committed:
Merge pull request #6408 from jia-kai/fix-search-ranking

Fix the ranking of search results

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
refactor: Separate generate_autosummary_content() from generate_autosummary_docs()

tk0miya committed:
Fix #247: autosummary: Add autosummary_generate_option to overwrite old stub file

tk0miya committed:
Merge commit '54d5fcfaebc3364044761d30c0fed6bd4d3052c3'

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Merge branch '2.0'

rhelmot committed:
Don't crash when autodocumenting classes with __slots__ = None

tk0miya committed:
Merge pull request #6448 from rhelmot/master

Don't crash when autodocumenting classes with __slots__ = None

tk0miya committed:
Merge branch '2.0'

vedranmiletic committed:
Add more EXAMPLES, update Blender and micca

tk0miya committed:
Fix flake8 violations

tk0miya committed:
Merge pull request #6469 from vedranmiletic/documentation-fixes

Add more EXAMPLES, update Blender and micca

jdufresne committed:
Remove unused import of InsecurePlatformWarning

Unused since 77e0139a26a080e5b7c0292ebd08fbb2eb540be9.

jdufresne committed:
Remove empty code block in sphinx.util.requests

Empty since 19a187cb5443808c284dd9e8f85ad13cba98182b.

If requests is installed with the security extras, they will be loaded
automatically by requests. The is no need to proactively load it. The
block simply did a "pass" in all cases.

tk0miya committed:
Merge pull request #6479 from jdufresne/unused

Remove unused import of InsecurePlatformWarning

tk0miya committed:
Merge branch 'master' into pkg-resource

tk0miya committed:
Merge pull request #6481 from jdufresne/pkg-resource

Remove empty code block in sphinx.util.requests

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Merge pull request #6490 from tk0miya/refactor_autosummary

refactor: Separate generate_autosummary_content() from generate_autosummary_docs()

tk0miya committed:
Fix #6512: typo in continuation label for tables

tk0miya committed:
Merge branch '2.0'

csabella committed:
Fix typo

tk0miya committed:
Merge pull request #6516 from csabella/patch-1

Fix typo in comment

tk0miya committed:
Merge pull request #6513 from tk0miya/6512_continuation_typo

Fix #6512: typo in continuation label for tables

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Fix flake8 violations

tk0miya committed:
Merge pull request #6539 from tk0miya/fix_flake8_violations

Fix flake8 violations

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Merge branch '2.0'

Blendify committed:
Opensearch: Use favicon if available

Blendify committed:
Opensearch: Use favicon if available

dullbananas committed:
Clarify Windows installation

tk0miya committed:
Merge pull request #6551 from dullbananas/master

Clarify Windows installation

tk0miya reviewed (changes requested):
LGTM. Could you rebase this into 2.0 branch? Then I'll merge this soon.

tk0miya committed:
Merge branch 'master' of github.com:sphinx-doc/sphinx

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Merge branch '2.0'

tk0miya committed:
Merge pull request #6422 from tk0miya/247_autosummary_autorewrite

Fix #247: autosummary: Add autosummary_generate_option to overwrite old stub file

tk0miya committed:
Merge branch '2.0'

TimKam committed:
#6503 document meta directive (#6562)

#6503 document meta directive

The `meta` directive has partly Sphinx-specific behavior
and hence needs to be documented

YuriPetrov committed:
Add "yuripetrov.ru/edu/python" to Examples page

tk0miya committed:
Merge pull request #6572 from YuriPetrov/master

Add "yuripetrov.ru/edu/python" to sphinx/EXAMPLES

tk0miya committed:
Merge branch '2.0'

Blendify committed:
Merge branch 'patch-1' of https://github.com/Blendify/sphinx into patch-1

Blendify commented:
Yikes did not mean to do that... Ugh...

tk0miya commented:
Merged manually.
Thank you for your contribution!
